ai_agents/agents/examples/voice-assistant-with-memU/tenapp/ten_packages/extension/main_python/memory.py

```
@@ -332,11 +335,11 @@ async def retrieve_related_clustered_categories(
                 ) as r:
                     r.raise_for_status()
                     result = await r.json()
-                    
+
                     self.env.log_info(
                         f"[MemuHttpMemoryStore] retrieve_related_clustered_categories returned successfully"
```

Ruff (0.14.0)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)